.github/workflows/scripts/slack_notifications/gem_notifier.rb

```diff
@@ -59,13 +59,24 @@ def self.gem_source_code_uri(gem_name)
     info = HTTParty.get("https://rubygems.org/api/v1/gems/#{gem_name}.json")
     raise "Response unsuccessful: #{info}" unless info.success?
 
-    info["source_code_uri"]
+    source_code_uri = info["source_code_uri"]
+    validate_github_url(source_code_uri)
+    source_code_uri
   rescue StandardError => e
     abort("#{e.class}: #{e.message}")
   end
 
+  def self.validate_github_url(url)
```

Comment:
Its possible that not all gems are source code points back to GitHub (GitLab could be one for example). It would be great to rename this method to be more broad.
```suggestion
  def self.validate_url(url)
```